### PR TITLE
feat: add FluxRecord.row with response data stored in List

### PR DESCRIPTION
**Pull request by michaelahojna**
Closes #

## Proposed Changes

Adding possibility of accessing response data in List FluxRecord.row.

In case of using pivot on data, where field contains labels that occur by default in the annotated CSV (f.e. "result" or "table"), could be duplicated column names in response. In that case FluxRecord.values (Map), which can hold only unique keys, doesn't show complete data. This edge case is solved by using FluxRecord.row (List).

## Checklist

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] A test has been added if appropriate
- [x] `pub run test` completes successfully
- [x] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)
- [x] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)

### CODE REVIEW

File: example/record_row_example.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    recordValues.add(record.values.join(","));
+    recordRow.add(record.row.join(", "));
+  });
+
+  print("-------------------------- record.values ---------------------------");
+  print(recordValues.join("\n"));
+  print("---------------------------- record.row ----------------------------");
+  print(recordRow.join("\n"));
+
+  await Future.delayed(Duration(seconds: 10));
```
Comment (bednar) [inline example/record_row_example.dart:50]: This is useless because the waiting is done by `await recordStream.forEach((record) {...`

```suggestion
```

**Review by bednar — CHANGES_REQUESTED** [review]
Thanks for your PR 👍 
There are a few requirements that must be be satisfy before we accept the PR:

**Review by bednar — APPROVED** [review]
LGTM 👍
